src/renderer/components/monaco-editor/monaco-editor.tsx

```diff
@@ -288,13 +291,19 @@ class NonInjectedMonacoEditor extends React.Component<MonacoEditorProps & Depend
   // avoid excessive validations during typing
   validateLazy = debounce(this.validate, 250);
 
+  get initialHeightClassName() {
+    return this.props.setInitialHeight ?
+      styles[this.props.getEditorHeightFromLinesCount(this.model.getLineCount())] :
+      null;
+  }
+
```

Comment:
Why this value needs to setup for css besides predefined css-names? (e.g. "large" from injectable).
Also probably better name required for the getter + add returned `type`.